xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-ui/src/main/resources/XWiki/Lightbox/WebHome.xml

```diff
@@ -469,7 +469,7 @@ bundles=blueimp-gallery,blueimp-helper,blueimp-gallery-fullscreen,blueimp-galler
       &lt;div class="descriptionRow"&gt;
         &lt;span class="publisher"
           aria-label="$escapetool.xml($services.localization.render('image.lightbox.publisher.label'))"&gt;&lt;/span&gt;
-        &lt;span class="date" data-dateFormat="$dateFormat"
+        &lt;span class="date" data-dateFormat="$dateFormat" data-userTimezone="$xwiki.getUserTimeZone()"
```

Comment:
small note: I'd write `data-userTimeZone` instead. It seems it takes a capital Z (e.g. https://docs.oracle.com/javase/7/docs/api/java/util/TimeZone.html).

Reply:
Note that uppercase letters have no meaning in HTML attributes (they are case-insensitive). You should use `data-user-time-zone` instead, which can also be requested in `jQuery` using `.data('userTimeZone')` or using `.dataset.userTimeZone` on a plain JavaScript object, see the [documentation for the dataset API](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset). Also, this is missing XML escaping for both the date format and the timezone.